CantEngine/Components/UIComponent.h

```diff
@@ -90,7 +91,14 @@ class UIComponent :	public BaseComponent
 	UIComponent* m_parent;
 	std::vector<UIComponent*> m_children;
 
-
+	std::string m_internalTextUI;
+	std::wstring m_textUI;
+	FontType m_textFontType;
+	//position should be defined from 0 to 1 (origin is top left of windows coordinate)
```

Comment:
No need to change anything. 
Noting this is a good example for what you should write in the doxygen @brief for any member if it needs some explanation